src/lib/y2users/config_manager.rb

```diff
@@ -81,5 +81,19 @@ def system(reader: nil, force_read: false)
 
       config
     end
+
+    # Gets target modification used to store configuration before it is applied
```

Comment:
NP: target modification? I don't understand it very well. I would rephrase it a bit: Config used to store the configuration to apply.